example-configuration.yml

```diff
@@ -0,0 +1,29 @@
+specification: specifications/petstore.yaml
+outputDir: example/generatedCode
+packageName: openapi
```

Comment:
Just FYI: You do (or at least should) not have to list every property in the configuration file (only the ones which deviate from the default).

Reply:
Yeah, I already thought so. :)
I just copied the `openapi-configuration.yml` file and made the necessary changes that seemed necessary. Not sure if these make sense. I can adapt the config to only contain the minimum necessary stuff.